README.md

```diff
@@ -35,6 +35,17 @@ Featured in the video:
 * Showing diagnostic message in a floating window.
 * Code actions.
 
+## How to get the most out of this plugin
+
+This plugin is designed to provide an opinionated and useable set of defaults for working with LSP and Completion in Neovim. A few things should be considered before you use it in your own config
+
+* To get the best experience, it is recommended to read through this page from top to bottom and to understand the options available to you. 
+* Many things that users often try to customize outside of this plugin are handled through the plugin functionality, and the "doubling up" of configuration can cause issues and loss of functionality. If you are using this plugin and want to customize anything, it is recommended to follow the instructions below and in the [Advance usage](https://github.com/VonHeikemen/lsp-zero.nvim/blob/main/advance-usage.md) page. 
+* Any setup code used outside the built in configuration options and related to LSP or Completion may result in undesired behavior. 
+* This plugin also handles many layers of abstraction in an effort to give a broad range of users the best possible experience out of the box. Some things you might be expecting might not be present as a result. If you feel something is missing, search the [discussions](https://github.com/VonHeikemen/lsp-zero.nvim/discussions) to see if it has already been discussed, and open a new discussion if not.  
+* If you want to set up completion yourself, you still can. See the [Available Presets](https://github.com/VonHeikemen/lsp-zero.nvim#available-presets) section for instructions on how to disable that portion of the plugin and assure your own setup does not conflict with the plugin
+* Snippet functionality is included by default via [Luasnip](https://github.com/L3MON4D3/LuaSnip) and a base collection of snippets are included for a variety of languages, but you can also add your own or add collections from other sources by following the Luasnip docs. 
```

Comment:
here it says "and a base collection of snippets are included".

We don't include snippets in lsp-zero. We recommend `friendly-snippets` in the quickstart section.

Reply:
> We don't include snippets in lsp-zero. We recommend friendly-snippets in the quickstart section.

Let's specify that explicitly then maybe? To me, it reads that `friendly-snippets` is one of the requirements from the way it's currently displayed.

Reply:
Clarified this in 6b13fd99ac3664357dafabd059e95c5ac21ef882